capability/capability_linux.go

```diff
@@ -383,6 +387,32 @@ func (c *capsV3) Apply(kind CapType) error {
 	return nil
 }
 
+func setAmbient(raise bool, cap ...Cap) error {
```

Comment:
nit: instead of `raise bool` you can use `action uintptr` and then call it like this:
```go
func ambientRaise(cap ...Cap) error {
	return setAmbient(pr_CAP_AMBIENT_RAISE, cap...)
}

func ambientLower(cap ...Cap) error {
	return setAmbient(pr_CAP_AMBIENT_LOWER, cap...)
}
```

This removes an if and makes the code more straightforward/direct.